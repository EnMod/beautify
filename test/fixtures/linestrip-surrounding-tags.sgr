p
  b a
  |
  | b
  |
  i c
