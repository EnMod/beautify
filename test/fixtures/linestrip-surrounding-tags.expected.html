<p><b>a</b>
b
<i>c</i></p>
